### PR TITLE
border

### LINKED ISSUE #527: [Feature]: border on code pill
### Description
Make the code pills have the nice boarder like leetcode.com

image

> [!IMPORTANT]
> ## this is needed fr fr ong

### PR DESCRIPTION
**Pull request by MVAodhan**
Add border to code pill. I spent way to long on this.

image

Closes #527

### CODE REVIEW

File: packages/email/yarn.lock
Comment (Hacksore) [inline packages/email/yarn.lock:1]: Delete this

**Review by Hacksore — CHANGES_REQUESTED** [review]
Screenshots of light and dark mode please

**Comment by Hacksore** [comment]
You also need to run prettier with `pnpm checks:fix`

File: packages/email/yarn.lock
Comment (MVAodhan) [reply packages/email/yarn.lock:1]: do i delete something?

**Comment by MVAodhan** [comment]
> 

image

image

File: packages/email/yarn.lock
Comment (QuiiBz) [reply packages/email/yarn.lock:1]: You can revert the changes by running `git checkout packages/email/yarn.lock main`

Comment (MVAodhan) [reply packages/email/yarn.lock:1]: > You can revert the changes by running `git checkout packages/email/yarn.lock main`

Thanks Tom, I tried running that and ran into an error.
image

I think it's and error I ran into when setting up codespaces. I just ended up deleting it like Hacksore said,

**Comment by bautistaaa** [comment]
we okay here now?

File: packages/email/yarn.lock
Comment (QuiiBz) [reply packages/email/yarn.lock:1]: Oh sorry it’s `git checkout main packages/email/yarn.lock`

**Comment by MVAodhan** [comment]
> we okay here now?

Waiting on @Hacksore

**Comment by bautistaaa** [comment]
please revert the changes on yarn lock

**Comment by MVAodhan** [comment]
> please revert the changes on yarn lock

I did a oopsy
image

**Comment by bautistaaa** [comment]
> > please revert the changes on yarn lock
> 
> I did a oopsy image

i cant teach you how to use github. if its easier just make a new PR

**Comment by MVAodhan** [comment]
@bautistaaa I think i fixed it right?

File: packages/ui/src/components/markdown.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ export function Markdown({ children, className }: { children: string; className?
               {String(children).replace(/\n$/, '')}
             </SyntaxHighlighter>
           ) : (
-            <code className="rounded-md bg-neutral-200 p-1 font-mono dark:bg-black">
+            <code className="rounded-md border border-gray-700 bg-neutral-200 p-1 font-mono dark:border-gray-700 dark:bg-black">
```
Comment (bautistaaa) [inline packages/ui/src/components/markdown.tsx:76]:
```suggestion
            <code className="rounded-md border border-gray-700 bg-neutral-200 p-1 font-mono dark:border-gray-300 dark:bg-black">
```

**Comment by bautistaaa** [comment]
> @bautistaaa I think i fixed it right?

i edited the tw rules. can you send updated SS

**Comment by MVAodhan** [comment]
> > @bautistaaa I think i fixed it right?
> 
> i edited the tw rules. can you send updated SS

image

image

**Comment by MVAodhan** [comment]
> > @bautistaaa I think i fixed it right?
> 
> i edited the tw rules. can you send updated SS

image

image

File: packages/ui/src/components/markdown.tsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ export function Markdown({ children, className }: { children: string; className?
               {String(children).replace(/\n$/, '')}
             </SyntaxHighlighter>
           ) : (
-            <code className="rounded-md bg-neutral-200 p-1 font-mono dark:bg-black">
+            <code className="rounded-md border border-gray-700 bg-neutral-200 p-1 font-mono dark:border-gray-300 dark:bg-black">
```
Comment (Hacksore) [inline packages/ui/src/components/markdown.tsx:76]: You don't want to make them look like buttons it should be subtle not high contrast windows theme looking.

cc @PickleNik

Light:
image

Dark:
image


```suggestion
            <code className="border dark:border-zinc-600 border-zinc-300 dark:text-zinc-300 text-zinc-600 rounded-md bg-neutral-200 p-1 font-mono dark:bg-zinc-700">
```

**Review by Hacksore — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/markdown.tsx (hunk 2 above)
Comment (MVAodhan) [reply packages/ui/src/components/markdown.tsx:76]: man basically at'ed the CSS speck @PickleNik

**Review by Hacksore — APPROVED** [review]

**Comment by maxdemaio** [comment]
@Tylopilus

> [!IMPORTANT]
> ## explain
> **Tylopilus** pushed a commit to Tylopilus/typehero that referenced this pull request [7 hours ago](https://github.com/typehero/typehero/pull/531#ref-commit-e526975)

**Comment by Hacksore** [comment]
> @Tylopilus
> 
> Important
> 
> ## explain
> **Tylopilus** pushed a commit to Tylopilus/typehero that referenced this pull request [7 hours ago](https://github.com/typehero/typehero/pull/531#ref-commit-e526975)

dw @MVAodhan, #531 was just referenced by @Tylopilus in his forked repo in a branch.

**Comment by MVAodhan** [comment]
> > @Tylopilus
> > Important
> > ## explain
> > **Tylopilus** pushed a commit to Tylopilus/typehero that referenced this pull request [7 hours ago](https://github.com/typehero/typehero/pull/531#ref-commit-e526975)
> 
> dw @MVAodhan, #531 was just referenced by @Tylopilus in his forked repo within a branch.

TY my dawg! ILY xx

**Pull request merged**